prow/cmd/branchprotector/BUILD

```diff
@@ -1,3 +1,44 @@
+# Usage:
+#   bazel run :dev-job.create
+load("@io_bazel_rules_k8s//k8s:object.bzl", "k8s_object")
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_bundle")
+load("@io_bazel_rules_docker//contrib:push-all.bzl", "docker_push")
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+
+k8s_object(
+    name = "dev-job",
+    cluster = "gke_k8s-prow_us-central1-f_prow",  # Run make :get-cluster-credentials
```

Comment:
Consider moving the cluster into a variable in `print-workspace-status.sh`, so that folks that don't have access to this cluster can deploy this.  `image_chroot` too, since randos can't push to `gcr.io/k8s-prow`.

Reply:
You can also hoist these options as defaults on a new macro (with a name of your choosing) in `WORKSPACE`.

For example (from `rules_k8s`):
```python
k8s_defaults(
    name = "k8s_deploy",
    cluster = _CLUSTER,
    image_chroot = "us.gcr.io/rules_k8s/{BUILD_USER}",
    # I'd actually avoid using kind, it gets you very little (.describe), and explodes the number of macros you need.
    kind = "deployment",
    namespace = _NAMESPACE,
)
```

I'll be in Seattle later, if you want to discuss.